src/main/java/io/vertx/ext/dropwizard/impl/HttpRequestMetric.java

```diff
@@ -12,10 +15,15 @@
   final String uri;
   HttpResponse response;
   long requestBegin;
+  final List<String> routes = new LinkedList<>();
```

Comment:
should't we keep here only a String and a single route ? or is this called multiple times by vertx web ?

Reply:
The call from vertx-web is made whenever a [`Route` matches](https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/impl/RoutingContextImplBase.java#L142). From my understanding this can occur more than once in the following circumstances:
* the initially matched route has no handlers attached, this will make routing context keep iterating, potentially finding another match. I consider this a quite unlikely situation and have to admit that concatenation of matched routes seems to be counter-intuitive here.
* the matched route is in fact a mounted subrouter which is going to perform route matching on its own. I think this might be a more common situation and concatenation of matched routes is desirable here (otherwise the base path would be missing)

I took the very same approach for the [micrometer implementation](https://github.com/vert-x3/vertx-micrometer-metrics/pull/111/files#diff-8ba7bb22b4edb4a9f37548bc1b86abe37f7f1bb6ef9014ef19e165c2ee8da223R120) but I am open for suggestions.

Reply:
ok I see, can we add a test for the case with multiple times ?

Reply:
sure. done.